### ISSUE TITLE
[FEATURE] Global skips

### ISSUE THREAD

**Issue by RoyalTS** [hadialqattan/pycln#62]
Is there a way to specify seemingly unneeded imports globally across all files? If not, could this be added, e.g. as a global config option?

**Comment by hadialqattan** [comment]
Hi @RoyalTS, I'm happy to see you around, anyway, to ensure that we're on the same page, I'm gonna give an example, and kindly let me know if I got you right:

You're looking for something more elegant than doing:
a.py:
```python3
import life  # nopycln: import
```

b.py:
```python3
import life  # nopycln: import
```

Like having a config option to skip some modules:
pyproject.toml:
```toml
[tool.pycln]
skip_imports = [life]
```

**Comment by hadialqattan** [comment]
@RoyalTS Kindly, I'm waiting for your response.

**Comment by GameDungeon** [comment]
I would like that. It would allow me to add it to my project. I currently have a constants file that expand_stars keeps unpacking. It unpacks the whole all file instead of just what it needs.

**Comment by GameDungeon** [comment]
Could I bump this? This is currently the deal breaker, but I would really like to use this :)

**Comment by hadialqattan** [comment]
Hi, is anybody still interested in this feature?  

If any, kindly hit the👍🏻reaction.

**Comment by hadialqattan** [comment]
> Could I bump this? This is currently the deal breaker, but I would really like to use this :)

Sorry for being too late, in case you still want to work on that, plz let me know to assign this issue to you and help you to finalize your PR.

**Comment by GameDungeon** [comment]
Your release reminded me of this project. I might spend some time seeing if I can get it to work for my project. Then I'll give you the working code in a pull :)

**Comment by hadialqattan** [comment]
> Your release reminded me of this project. I might spend some time seeing if I can get it to work for my project. Then I'll give you the working code in a pull :)

I'm excited to see your PR...please feel free to ask for any help if needed.

**Comment by hadialqattan** [comment]
Hi @GameDungeon, I've implemented the feature, kindly, have a look at PR #121 and let me know If you have any suggestions (especially regarding the docs/helping message). 

- [docs](https://github.com/hadialqattan/pycln/pull/121/files#diff-0b5ca119d2be595aa307d34512d9679e49186307ef94201e4b3dfa079aa89938)
- [helping message](https://github.com/hadialqattan/pycln/pull/121/files#diff-f28ec101cc5960df9ed5c806d60bf72b391aacab8b4e727d446825bb55f5cfc5)

Best,
